README.md

```diff
@@ -1,9 +1,26 @@
 # cloudfixLinter-demo-cloudformation
 
-For Demo Testing 
- 1. Install the extension
- 2. Make sure that a stack is already deployed to your AWS account
- 3. AWS account credentials are required in the default profile in `~/.aws/credentials`
+## Using Extension with Cloudformation
+ 1. Login to AWS using terminal by any of the AWS login methods:
+    1. `aws configure` to setup your AWS account (in default profile)
+    2. Using `saml2aws`
+    3. Set and export `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` as enviroment variables.
+ 2. Deploy a stack in your AWS account using the template (No need to do this again if done once and that stack isn't deleted). There are two ways to do this:
+    1. Use AWS [console](https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks)
+    2. Login to aws through CLI, then run this command
```

Comment:
doesn't the subnet need to be overwritten before this?

Reply:
Step added